tests/editing/totals_and_components/test_totals_and_components.py

```diff
@@ -6399,15 +6595,17 @@ def calculate_weighted(total: float, components: List[float]):
     print(f"Component sum is: {component_sum}")
 
     for i, component in enumerate(components):
-        adjusted_components.append((Decimal(component) / component_sum) * Decimal(total))
+        adjusted_components.append(
+            (Decimal(component) / component_sum) * Decimal(total)
```

Comment:
The values are already Decimal when we get here

Reply:
Changed